src/frontend/qt_sdl/main.cpp

```diff
@@ -2890,40 +2894,29 @@ int main(int argc, char** argv)
     Input::OpenJoystick();
 
     mainWindow = new MainWindow();
+    if (CLI::StartOnFullscreen) {
```

Comment:
braces go onto the next line

Reply:
missed that one somehow :sweat_smile:
fixed!